### DISCUSSION TITLE
new assets presets

### DISCUSSION THREAD

**Discussion by nikodiseas** [jetelain/ArmaRealMap#57] [General]
so far, only the Sahel and central Europe exist, and it is very problematic to create maps from other areas. will there be new presets? can't users share their presets? it would be very interesting to create Mediterranean maps. tnks

**Comment by OttoSuwen** [comment]
Hey I'm not the dev but it is detailed on how you can make your own preset (Here:https://github.com/jetelain/ArmaRealMap/wiki/Create-an-asset-configuration-for-Arma-3) also you can save your presets so essentially yeah you could probably share preset's

**Comment by jetelain** [comment]
Hello,

You can make your own presset, but to get something really different from CentralEurope and Sahel it may be diffucult, because you will have to get your own texture and create related config in Arma (not included in the tool for now).

I've planned to create a Mediterranean and Trocpical preset (based on Altis and Tanoa assets). I will work on it once I've added support for islands / ocean  (see issue #52).

**Comment by jetelain** [comment]
It could be great to have an easy way to share presets, I may create later a web site for that purpose.

**Comment by nikodiseas** [comment]
good. I understand you've practically made islands, haha. when will there be an update? another question concerns dlc from community. I don't see them in the search for objects, although I bought them all. why?

**Reply by jetelain** [reply]
Update should be available Wednesday, it already have been submitted to the store. For C DLC, it's not yet possible.

**Reply by nikodiseas** [reply]
could you explain how I can add my texture to the necessary configs? the fact is that when approaching the ground with my texture, it turns black. I tried to create Greece, the sahara myself, but alas.
